### ISSUE TITLE
Llava 1.5 doesn't work with transformers 4.39.0

### ISSUE THREAD

**Issue by xwjiang2010** [huggingface/transformers#29789]
### System Info

- `transformers` version: 4.39.0
- Platform: Linux-6.2.0-1017-aws-x86_64-with-glibc2.31
- Python version: 3.9.18
- Huggingface_hub version: 0.21.2
- Safetensors version: 0.4.2
- Accelerate version: 0.27.2
- Accelerate config:    not found
- PyTorch version (GPU?): 2.1.2+cu121 (True)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

```
from transformers import LlavaForConditionalGeneration
model = LlavaForConditionalGeneration.from_pretrained("llava-hf/llava-1.5-7b-hf")
```

### Expected behavior

run through without error

**Comment by amyeroberts** [comment]
Hi @xwjiang2010, thanks for flagging! 

We'll merge in a fix + patch release soon

**Comment by NielsRogge** [comment]
Closing as this is resolved by #29389
